tools/travis_before_install.sh

```diff
@@ -64,7 +63,7 @@ virtualenv -p python ~/venv
 source ~/venv/bin/activate
 
 python -m pip install --upgrade pip
-pip install --retries 3 -q wheel flake8 codecov nose
+pip install --retries 3 -q wheel flake8 codecov pytest pytest-cov
```

Comment:
Well… Reading this line, I am realizing that it is a bit strange that coverage isn't here. I'll check this out.